src/pages/developers/apps/intro.mdx

```diff
@@ -43,17 +43,108 @@ contract YourContract is zContract {
 }
 ```
 
-It’s implemented in Solidity and uses ZetaChain's universal EVM features (like
-`onCrossChainCall`) to handle calls from connected chains.
+## Calling Universal Apps
 
-<Alert variant="tip">
-  For a quick hands-on introduction to building omnichain apps check out "[Your First Universal
-  App](/developers/tutorials/hello)" tutorial.
-</Alert>
+Users can call universal apps by interacting with a **Gateway contract** on a
+connected chain. Each connected chain has a single Gateway contract that exposes
+methods to deposit tokens to and call universal apps. Users can pass both data
+and tokens when calling universal apps.
 
-One of the key advantages of a universal app is that it requires deploying only
-a single contract on ZetaChain, eliminating the need for separate contracts on
-each connected chain. This reduces development time, lowers deployment costs,
-and minimizes the potential for errors. You can focus on creating robust,
-feature-rich applications without worrying about the complexities of managing
-multiple contracts across different blockchains.
+<Alert>Gateway is an upcoming unified interface that will replace the TSS address and ERC-20 custody contract.</Alert>
```

Comment:
put tentative version number when this is going to be alive?
I think it's going to be v19, mainnet by end of Q3)

Reply:
https://github.com/zeta-chain/docs/pull/387/commits/23abf10441651f1607debc83acbd3e1de2232cf5